File: ct_206/Project 4/write_list_to_file.py
# Opens file list.txt and allows write permissions
# Returns file object
def open_file():
  f = open('list.txt', 'w+', encoding='utf-8')
  return f

# Writes provided list object to list.txt
# File object and list of data are the only parameters
def write_to_file(f, list):
  for item in list:
    f.write(str(item) + '\n')

# Closes file i/o
# File object is the only provided parameter
def close_file(f):
  f.close()


list = ['a', 'b', 'c', 1, 2, 3, 'x4y5z6']
f = open_file()
write_to_file(f, list)
close_file(f)

File: ct_206/Project 4/write_list_to_file.pl
@list = ('perl', 'b', 'c', 1, 2, 3, 'x4y5z6');

# Open new file to write
open(f1, ">list.txt") or die "Could not open file!";

# Write data from list to file
for (my $i = 0; $i < @list; $i += 1) {
  print f1 @list[$i];
  print f1 "\n";
}

close(f1);